apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -69,23 +111,84 @@ const ChatWorkspace: React.FunctionComponent<ChatWorkspaceProps> = ({
     }
   };
 
+  const iconValue: FontAwesomeV6IconProps = {
+    iconName: 'lock',
+    iconStyle: 'solid',
+  };
+
+  const tabs = [
+    {
+      value: 'viewStudentChatHistory',
+      text:
+        `${selectedStudentName}'s chat history` +
```

Comment:
Hey @sanchitmalhotra126, just a FYI that I updated this as requested by Mark Barrett after your most recent review.